src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The `alive` array is used to track all instances of `Animal` that are alive. Ensure that this array is correctly updated whenever an animal's health changes or when a new animal is created.